Tests/SwiftFrameTests/ImageLoaderTests.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+import XCTest
+@testable import SwiftFrameCore
+
+class ImageLoaderTests: XCTestCase {
+
+    func testLoadImage() throws {
+        let rep = makeImageRepresentationWithSize(.square100Pixels)
+        guard let cgImage = rep.cgImage else {
+            throw NSError(description: "Could not make CGImage from Bitmap")
```

Comment:
You can use `XCTFail` here. Alternatively you can also use `XCTUnwrap` to avoid the `guard`.